### PR TITLE
Added Date Tile

### PR DESCRIPTION
**Pull request by Keitech**
Screen Shot 2020-11-29 at 2 35 58 PM

Added a Date tile that displays current date and current time. While the current time updates every second

### CODE REVIEW

File: src/components/Dashboard/DateTile.tsx
Comment (dannyshwan) [inline src/components/Dashboard/DateTile.tsx:53]: lgtm. 

A nice to have though is to have this done as a functional component instead of a class component; mainly for consistency and simplicity.

**Review by dannyshwan — APPROVED** [review]

**Comment by dannyshwan** [comment]
Don't merge until one of harsh or bhala approves.

File: src/components/Dashboard/DateTile.tsx
Comment (guptaharsh25) [inline src/components/Dashboard/DateTile.tsx:24]: can the stylings start going in a separate file called `styled.tsx`?
this will help keep things organized.

Comment (guptaharsh25) [reply src/components/Dashboard/DateTile.tsx:24]: see discord advise on organization

File: src/views/Dashboard.tsx
Comment (guptaharsh25) [inline src/views/Dashboard.tsx:10]: This is fine for now, but I don't think this `views` folder makes sense, I think this file should go inside `components/Dashboard`

**Review by guptaharsh25 — CHANGES_REQUESTED** [review]

File: src/components/Dashboard/DateTile.tsx
Comment (Keitech) [reply src/components/Dashboard/DateTile.tsx]: moved to separate file. Going to reorganize everything in another pr

**Review by guptaharsh25 — APPROVED** [review]

**Pull request merged**
